e2e/tests/components/date-time-picker/date-time-picker.e2e-spec.ts

```diff
@@ -182,4 +182,20 @@ describe('Date Time Picker Tests', () => {
         expect(await page.getDisabled(await page.getCalendarItem('2019'))).toBe(false);
 
     });
-});
\ No newline at end of file
+
+    it('should correctly set the date when the current day value does not exist in the selected month', async () => {
+        // select 31st January
+        let date = await page.getDate('31');
+        await date.click();
+
+        // select 1st February
+        date = await page.getDate('1', true);
+
+        if (await page.isClickable(date)) {
```

Comment:
Is the purpose of this because in the MF theme future and previous month buttons are invisible?

![image](https://user-images.githubusercontent.com/20795331/89650202-f3db3400-d8b9-11ea-99d6-5789179b42ce.png)

Reply:
Yep

Reply:
Could we perhaps add a comment here describing the reason as it is not immediately obvious. Also perhaps the `isClickable` function might be better called `attemptClick` as `isClickable` suggests it is a getter with no side-effects when it is not so much checking if the element is clickable but performing a click and returning a success state

Reply:
Good point, I've made those changes to make it more clear 👍